src/quo2/foundations/typography.cljs

```diff
@@ -1,26 +1,25 @@
 (ns quo2.foundations.typography)
 
-;; Formulat for letter spacing from figma %: 16/{figma-percentage)*100
```

Comment:
this can't be the formula for letter-spacing, as it is inversely proportional,
when there should be more spacing it will give less, when there should be less like 0, it will give infinite

Reply:
@Parveshdhull [Figma](https://forum.figma.com/t/letter-spacing-should-not-be-percentage-based/3062) can also express the letter spacing in `em` rather than percentages. The body text having a spacing of `-0.003em` cc @pedro-et

Reply:
hi @hesterbruikman, Thank you very much for the link, I will explore the program/formula and see how it can be used.